cf/docstring/docstring.py

```diff
@@ -80,6 +80,14 @@
         Whether `esmpy` logging is enabled or not is determined by
         `cf.regrid_logging`. If it is enabled then logging takes place
         after every call. By default logging is disabled.""",
+    # subspace halos
+    "{{subspace halos}}": """
+        If a halo is defined via a positional argument, then each
+        subspaced axis will be extended to include that many extra
```

Comment:
Quite minor of an issue, but the text is getting, post-processing, two blank lines to break it up from the heading with this in use (e.g. see `help(f. indices)` for some Field `f`) so I guess you have to start the text straight after the triple quote to avoid this with the docstring substitution, e.g:

```suggestion
    "{{subspace halos}}": """If a halo is defined via
        a positional argument, then each subspaced axis
        will be extended to include that many extra
```